src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -117,16 +116,35 @@ export class VegaParser {
     if (this.isVegaLite !== undefined) throw new Error();
 
     if (typeof this.spec === 'string') {
-      this.spec = hjson.parse(this.spec, { legacyRoot: false });
+      const spec = hjson.parse(this.spec, { legacyRoot: false });
+
+      if (!spec.$schema) {
+        throw new Error(
+          i18n.translate('visTypeVega.vegaParser.inputSpecDoesNotSpecifySchemaErrorMessage', {
+            defaultMessage: `Your specification requires a {schemaParam} field with a valid URL for
```

Comment:
Markdown does not seem to be parsed here, so there is no use in using markdown here, we instead should just make this a regular text:

> Your specification requires a {schemaParam} field with a valid URL for Vega (see {vegaSchemaUrl}) or Vega-Lite (see {vegaLiteSchemaUrl}). The URL is an identifier only. Kibana and your browser will never access this URL.